com.unity.shadergraph/Editor/Drawing/ViewModels/InspectorViewModel.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+namespace UnityEditor.ShaderGraph.Drawing
+{
+    class InspectorViewModel : ISGViewModel
+    {
+        public GraphData Model { get; set; }
+
+        public VisualElement parentView { get; set; }
+
+        public void Reset()
+        {
```

Comment:
Nothing to clear here?

Reply:
Will add the other members to be cleared here, but in truth this is mostly a stub class right now as the Inspector has no controller, it's just that as a `GraphSubWindow` it now requires a view model of some sort so this is a minimal class to satisfy that requirement.